lib/formats/kms.js

```diff
@@ -0,0 +1,288 @@
+// Copyright 2017 Joyent, Inc.
+
+module.exports = {
+	listKeys: listKeys,
+	read: read,
+	verify: verify,
+	sign: sign,
+	write: write,
+	toBuffer: toBuffer,
+};
+
+var assert = require('assert-plus');
+var SSHBuffer = require('../ssh-buffer');
+var crypto = require('crypto');
+var Buffer = require('safer-buffer').Buffer;
+var algs = require('../algs');
+var Key = require('../key');
+var ecdsa = require('./pkcs1')
+var PrivateKey = require('../private-key');
+var Identity = require('../identity');
+var rfc4253 = require('./rfc4253');
+var Signature = require('../signature');
+var utils = require('../utils');
+var Certificate = require('../certificate');
+var asn1 = require('asn1')
+const { KMSClient, ListKeysCommand, GetPublicKeyCommand, SignCommand } = require("@aws-sdk/client-kms");
+const pkcs1 = require('./pkcs1');
+
+function verify(cert, key) {
+	/*
+	 * We always give an issuerKey, so if our verify() is being called then
+	 * there was no signature. Return false.
+	 */
+	return (false);
+}
+
+let TYPES = {
+	'user': 1,
+	'host': 2
+};
+Object.keys(TYPES).forEach(function (k) { TYPES[TYPES[k]] = k; });
+
+let ECDSA_ALGO = /^ecdsa-sha2-([^@-]+)-cert-v01@openssh.com$/;
```

Comment:
```suggestion
const ECDSA_ALGO = /^ecdsa-sha2-([^@-]+)-cert-v01@openssh.com$/;
```

This is never re-assigned from what I can see